Upgrade PostMessage to support multipart messages.

Daniel1898 opened the pull request:
Add MediaTypeUtils and FileController to download files from multipart messages.
Add method getFileFromMessage in ImapConnector to get file from imap server.

ValentinBodrov commented on Spring-server/src/main/java/ikpi63holding/postclient/ApiController.java:
```diff
@@ -117,6 +117,7 @@ void deleteUser(@PathVariable Long id) {
         }
     }
 
+
```
nice string :DDDD

IReallyHateLogins committed:
Merge branch 'develop' into multipart-messages-support

ValentinBodrov commented on Spring-server/src/main/java/ikpi63holding/postclient/ImapConnector.java:
```diff
@@ -9,13 +10,8 @@
-import javax.mail.PasswordAuthentication;
-import javax.mail.Session;
-import javax.mail.Store;
+import javax.mail.*;
```
I think it'll be better if we use more "java.mail.Something" instead of "java.mail.*"

ValentinBodrov commented on Spring-server/src/main/java/ikpi63holding/postclient/PostMessage.java:
```diff
@@ -1,20 +1,15 @@
+import java.io.InputStream;
+import java.util.*;
+import javax.mail.*;
+import javax.mail.internet.*;
```
Same thing with wildcards

Egridnev replied:
@Daniel1898 you have to use full path to insert only necessary imports

ValentinBodrov reviewed (commented):
Great job with little things to fix

Egridnev commented on Spring-server/src/main/java/ikpi63holding/postclient/FileController.java:
```diff
@@ -0,0 +1,50 @@
+                // Content-Type
+                .contentType(mediaType)
+                // Contet-Length
+                //.contentLength(file.length()) //
```
@Daniel1898  why did you comment this parameter? if we dont need it for future, maybe it`s better to remove it from code?

Daniel1898 replied:
Because I don't know how to get length from stream. But may be we can do it in the future.

IReallyHateLogins replied:
Mark it as TODO later, and I think it's fine

Egridnev reviewed (changes requested):
Point out some questions

Commit:
Upgrade PostMessage to support multipart messages.
Add MediaTypeUtils and FileController to download files from multipart messages.
Add method getFileFromMessage in ImapConnector to get file from imap server.

IReallyHateLogins committed:
Some repository and styling fixes

IReallyHateLogins reviewed (approved):
Well done!
